core/src/main/java/com/github/steanky/element/core/key/PathKeySplitter.java

```diff
@@ -0,0 +1,25 @@
+package com.github.steanky.element.core.key;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Splits "path keys" into string arrays.
+ */
+public interface PathKeySplitter {
```

Comment:
can you call it a key splitter if it doesn't split keys

Reply:
changed in most recent commit